plugins/importexport/native/filter/NativeFilterHelper.inc.php

```diff
@@ -284,11 +378,93 @@ function parseIssueCover($filter, $node, $object) {
 
 						file_put_contents($filePath, base64_decode($n->textContent));
 						break;
+					case 'href':
```

Comment:
This is similar to the previous modified code block, so the same comments can be applied